libs/ui/src/lib/pagination/index.tsx

```diff
@@ -10,10 +12,18 @@ interface Props {
 }
 
 const Pagination = ({ onPageChange, totalPages, currentPage }: Props) => {
+  const searchParams = useSearchParams();
+  const urlPage = searchParams.get('page') ? Number(searchParams.get('page')) : currentPage;
```

Comment:
Ikke legg det i pagination componenten. Den kan eventuelt brukes av andre og flere og da kan det være at den bruker andre params

Reply:
Bra innspill. 👍🏼  Jeg er litt uenig i at vi i monorepoet skal begynne å ta høyde for at komponentene skal brukes av andre løsninger utenfor monorepoet. Prøvde å brainstorme litt på frontendkanalen på slack.